layouts/partials/hero.html

```diff
@@ -0,0 +1,6 @@
+<section class="hero">
+  {{ partial "navbar.html" . }}
+  {{ partial "navbar-clone.html" . }}
+  {{ partial "hero-body.html" . }}
+  {{ partial "hero-footer.html" . }}
+</section>
```

Comment:
Can we rename the ` hero.html` file name according to numpy

Reply:
This is again only showing up on the homepage. The above the fold section of the homepage is usually called the `hero` or `jumbotron`, so this is yet another convention. We can rename it, but this should only occur one time on any given site.

Reply:
Makes sense to not change theme naming conventions like `hero` or `section1`.